inspectit-oce-documentation/src/docs/asciidoc/_includes/configuration/configuration-basics.adoc

```diff
@@ -0,0 +1,69 @@
+=== Configuration Basics
+
+inspectIT OCE tries to implement the zero-configuration approach but lets you externalize the configuration and influence every bit of its configuration if this is required.
+Internally it uses the Spring-based `PropertySource` order to allow overriding of configuration values.
+Configuration properties are considered in inspectIT OCE in the following order of priority:
+
+. <<Java System properties,Java System properties>>
+. <<OS Environment Variables,OS environment variables>>
+. Supported <<Externalized Configuration Methods,externalized configuration methods>>
+.. <<File-based Configuration,File-based configuration>>
+. <<All Configuration Options,inspectIT OCE defaults>>
+
+The overview of all available inspectIT OCE configuration properties is given in the <<All Configuration Options>> section.
+
+==== Java System Properties
+
+You can pass any configuration property as the Java System property to the Java command that you use to start your Java application.
+For example, to override the service name used to identify your application reporting the performance data, you can change the `inspectit.service-name` property:
+
+[subs=attributes+]
+```bash
+$ java -Dinspectit.service-name=my-custom-service -javaagent:inspectit-oce-agent-{inspectit-oce-version}.jar -jar my-java-program.jar
```

Comment:
Maybe use quotation marks and a name with spaces to show how to deal wit hspaces when passing properties?